### ISSUE TITLE
[FEATURE] Move 'trans' template tag to 'translate' in django templates

### ISSUE THREAD

**Issue by frague59** [djlint/djLint#770]
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Feature Request

From django docs, I can see the `trans` template tag has been renamed to `translate` from django 3.1.

https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#translate-template-tag

The `blocktrans` has been moved to `blocktranslate` too.

Is it possible djLint warn users about these namings ?

Thanks! 🤠

**Comment by ronanboiteau** [comment]
Hey @frague59! If you have already refactored your code base to use the new names, you can use a `.djlint_rules.yaml` file to create rules to find the outdated tags and trigger errors. Something like this:

```yaml
- rule:
    name: CR01
    message: Deprecated template tag 'trans' found. Use 'translate' instead.
    flags: re.DOTALL
    patterns:
      - \{% *trans +
- rule:
    name: CR02
    message: Deprecated template tag 'blocktrans' found. Use 'blocktranslate' instead.
    flags: re.DOTALL
    patterns:
      - \{% *blocktrans +
- rule:
    name: CR03
    message: Deprecated template tag 'endblocktrans' found. Use 'endblocktranslate' instead.
    flags: re.DOTALL
    patterns:
      - \{% *endblocktrans *%}
```

More details about custom rules: https://djlint.com/docs/linter/

**Comment by frague59** [comment]
Thanks for your answer, I'll put this into my projects.
